concent-builder/update-nginx-proxy-on-the-second-cluster.yml

```diff
@@ -0,0 +1,70 @@
+- hosts:
+    - concent-builder
+  vars_files:
+    - consts.yml
+    - repositories.yml
+    - "{{ deployment_values }}/var.yml"
+    - "{{ deployment_values }}/var-{{ cluster }}.yml"
+  tasks:
+    - become:      yes
+      become_user: "{{ shared_user }}"
+      block:
+        - name: Clone concent repositories
+          git:
```

Comment:
We should talk about this tomorrow.

Reply:
1. Please remove the container build from this playbook. Built images of previous tagged versions should already be available in our docker registry and can be rebuilt at any time anyway.
2. Running the playbook removes the configuration built by `build-test-and-push.yml` so `deploy.yml` won't work as expected. This might be unexpected. Maybe it would be better to build in a separate directory? E.g. `"{{ build_dir }}/concent-deployment-<deployment version>"`?
3. Please rename it to `redeploy-nginx-proxy-router.yml`.

This new playbook does not really fit well into the set of playbooks we've built so far. We'd have to reorganize the other playbooks to make it more consistent, though I don't yet have a clear idea how. I don't like that that container build is not a separate stage and that kubernetes configuration just leaves files in the build directory instead of creating a package.

But this is not a good time to change that so the changes listed above will have to suffice for now.

Reply:
Let's call the directory `concent-deployment-nginx-proxy`.